csrc/index_compute.cpp

```diff
@@ -3878,38 +3885,98 @@ std::pair<Val*, Val*> Index::getCpAsyncBulkGmemIndex(
 
   int64_t itemsize = dataTypeSize(gmem_tv->dtype());
 
+  // Loop through the frontier from inner to outer, handle TMA-global ID and
```

Comment:
Could you also give a simple illustrative example?

Reply:
Added a few examples in https://github.com/NVIDIA/Fuser/pull/2004/commits/43a43d181144417b52c94449d1195320142e8961
Example 1 and 2 are relavent to this PR.